### PR TITLE
[WIP] Add close method

### PR DESCRIPTION
**Pull request by simison**
This might be useful when writing tests.

[See example](https://github.com/Trustroots/trustroots/blob/master/gulpfile.js#L402) at Trustroots.

Replaces similar PR #166

Any pointers how to get rid of this warning:
```
      close
        ✓ closes database connection
        ✓ returns itself
(node:84221) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): MongoError: topology was destroyed
(node:84221) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): MongoError: topology was destroyed
```

I'm ensuring `close()` is always called with callback function so it shouldn't return promises:
```js
this._mdb.close(force, cb || function(){});
```

Docs for close: https://mongodb.github.io/node-mongodb-native/2.0/api/Db.html#close

### CODE REVIEW

**Review by michelem09 — DISMISSED** [review]
LGTM

**Comment by simison** [comment]
I'll need to rebase this.

**Comment by dandv** [comment]
Haven't looked into this yet, but I've seen Agenda throw 'topology was destroyed' if it attempted Mongo operations after the connection was already closed elsewhere.

**Comment by rdzak** [comment]
Hi! Any plan of incorporating it? Seem like basic functionality to me. My mocha tests are hanging because I cannot close this connection.

**Review by rdzak — DISMISSED** [review]

**Comment by koresar** [comment]
May I insert my 5 cents?
I'd recommend people to read about "magic boolean problem": https://www.google.com/search?q=magic+boolean+problem

TL;DR: I'd suggest changing the API from `db.close(true)` to `db.close({ force: true })`.

**Comment by simison** [comment]
@koresar I agree, valid point!

**Review by leonardlin — APPROVED** [review]

**Comment by simison** [comment]
Remember to update the changelog, folks!

**Comment by leonardlin** [comment]
oh yes will do
